docs/scos/dev/front-end-development/202212.0/oryx/oryx-feature-sets.md

```diff
@@ -26,6 +26,67 @@ Oryx includes predefined feature sets that cover common use cases for web applic
 - b2cFeatures: features commonly used in B2C applications.
 - fulfillmentFeatures: features used in PWAs, used for picking products for fulfillment.
 
+## Static Experience Data
+
+Feature sets also contain (static) experience data. Experience data includes the structure and layout of the components of the application. Feature sets provide all the static pages data that is required for the application experiences. The `b2cFeatures` feature set, for example, comes with the following static data:
```

Comment:
`Feature sets provide all the static pages data that is required for the application experiences.`

What new information does this sentence add?

Reply:
It's the first time that we talk about "pages". Pages will have more attention in the docs going forward, but it is partially covered by static experience data.